package.json

```diff
@@ -2,73 +2,74 @@
   "name": "root",
   "license": "2-clause BSD license plus a third clause that prohibits redistribution and use for commercial purposes without further permission.",
   "engines": {
-    "node": "^14.4",
-    "npm": "7.x"
+    "node": "16.x",
+    "npm": "8.x"
   },
+  "workspaces": [
+    "packages/desktop",
+    "packages/web"
+  ],
   "private": true,
   "scripts": {
-    "bootstrap": "lerna bootstrap --hoist",
     "lint": "eslint --ext .jsx --ext .js --ext .tsx --ext .ts packages/",
-    "test:core": "cd packages/core && BABEL_ENV=nodejs mocha",
+    "test:core": "cd packages/core && BABEL_ENV=nodejs mocha --exit",
     "test:desktop": "cd packages/desktop && npm test",
-    "test:web": "cd packages/web && BABEL_ENV=nodejs mocha",
+    "test:web": "cd packages/web && BABEL_ENV=nodejs mocha --exit",
     "test": "npm-run-all test:core test:desktop test:web",
     "typeCheck": "tsc --noEmit"
   },
   "devDependencies": {
-    "@babel/cli": "7.13.x",
-    "@babel/core": "7.13.x",
-    "@babel/plugin-proposal-class-properties": "7.13.x",
-    "@babel/plugin-proposal-optional-chaining": "7.13.x",
-    "@babel/preset-env": "7.13.x",
-    "@babel/preset-react": "7.12.x",
-    "@babel/preset-typescript": "7.13.x",
-    "@babel/register": "7.13.x",
-    "@testing-library/react": "11.1.x",
+    "@babel/cli": "7.x",
```

Comment:
Is babel still needed for TypeScript in 2022? Can't you just the TypeScript compiler?

Reply:
Arguably `ts-loader` could have been used instead of `babel-loader` for years now. If I'm remembering correctly, however, I've been using `babel` instead because other parts of the development toolchain use it (like testing), so I figured it was better to only use one tool where possible, and I believe this follows the lead of `create-react-app`.